### ISSUE TITLE
Wrapper to have multiple plugins on one screen?

### ISSUE THREAD

**Issue by fohria** [jspsych/jsPsych#621]
Thanks for jsPsych, it's really clean and great documentation.

I created my own plugin that combines image-button-response and the survey-likert plugins, because I wanted a likert scale on the same screen as the image and button.

Now I wonder, would it be possible to create a "wrapper" plugin, that takes as parameters other plugins, to combine plugins more dynamically? Or would that require rewriting core parts of the library?

**Comment by jodeleeuw** [comment]
I think that would require some pretty heavy rewrites of both the core and
the plugins. Each plugin assumes that it gets complete control over the
display element, so there's no real way for plugins to share control.

**Comment by fohria** [comment]
oh okay, i suspected that was the case, too bad. it would be really neat as a future improvement :)

**Comment by Cerebrock** [comment]
Yeah I think this is quite essential, if one has a small questionnaire that asks for age, gender, and work context, it doesn't make much sense to have 3 screens for text input, multi choice and multi select for example. 
Other than that thank you for the contrib! Awesome tool
